File: JavaScript/Basics/0.1-First-Steps-In-Coding/Exercise/fruit-market.js
function billCalculator(input) {
    let strawberriesPrice = Number(input[0]);
    let bananasCount = Number(input[1]);
    let orangesCount = Number(input[2]);
    let raspberriesCount = Number(input[3]);
    let strawberriesCount = Number(input[4]);
    let raspberriesPrice = strawberriesPrice / 2;
    let orangesPrice = orangesCount * (raspberriesPrice - raspberriesPrice * 0.40);
    let bananasPrice = bananasCount * (raspberriesPrice - raspberriesPrice * 0.80);
    raspberriesPrice *= raspberriesCount;
    strawberriesPrice *= strawberriesCount;
    let result = raspberriesPrice + strawberriesPrice + orangesPrice + bananasPrice;
    console.log(result);
}

billCalculator(["48", "10", "3.3", "6.5", "1.7"])
billCalculator(["63.5", "3.57", "6.35", "8.15", "2.5"])

File: Python/0.1-Basics/0.1-First-Steps-In-Coding/Exercises/fruit_market.py
strawberries_price = float(input())
bananas_kg = float(input())
oranges_kg = float(input())
raspberries_kg = float(input())
strawberries_kg = float(input())

raspberries_price = strawberries_price / 2
oranges_price = raspberries_price - (raspberries_price * 0.40)
bananas_price = raspberries_price - (raspberries_price * 0.80)

total_price = strawberries_kg * strawberries_price
total_price += bananas_kg * bananas_price
total_price += oranges_kg * oranges_price
total_price += raspberries_kg * raspberries_price
print(total_price)
